script.js

```diff
@@ -0,0 +1,43 @@
+window.addEventListener('load', e => {
+
+    const monthlyPriceBasic = 19.99;
+    const monthlyPriceProfessional = 24.99;
+    const monthlyPriceMaster = 39.99;
+
+    const yearlyPriceBasic = 199.99;
+    const yearlyPriceProfessional = 249.99;
+    const yearlyPriceMaster = 399.99;
+    
+    const planPriceBasic = document.querySelector('.basic-content .plan-price');
+    const planPriceProfessional = document.querySelector('.professional-content .plan-price');
+    const planPriceMaster = document.querySelector('.master-content .plan-price');
```

Comment:
Siempre ten en mente que el codigo lo escribes para personas, nombra los espacios en memoria con eso en mente, te sugiero esto:

```suggestion
    const basicPlanHTMLElement = document.querySelector('.basic-content .plan-price');
    const proPlanHTMLElement = document.querySelector('.professional-content .plan-price');
    const masterPlanHTMLElement = document.querySelector('.master-content .plan-price');
```